pom.xml

```diff
@@ -314,7 +314,7 @@
         <jar.plugin.version>2.6</jar.plugin.version>
         <javadoc.plugin.version>2.10.2</javadoc.plugin.version>
         <pax.plugin.version>1.5</pax.plugin.version>
-        <reflections.plugin.version>0.9.8</reflections.plugin.version>
+        <reflections.plugin.version>0.9.9-RC2</reflections.plugin.version>
```

Comment:
I'm generally against depending on RC versions, can we leave this one behind?

Reply:
let me see if there are any problems sticking with 0.9.8 plugin while bumping the core dependency to 0.9.11.  

"org.reflections" % "reflections-maven" % "0.9.8" is from 24-May-2012, the primary reasoning to for 0.9.9-RC2 is it dropped 13-Jun-2014


--




http://search.maven.org/#search%7Cgav%7C1%7Cg%3A%22org.reflections%22%20AND%20a%3A%22reflections-maven%22

Reply:
digging into this further, I visited https://github.com/ronmamo/reflections-maven

which notes that reflections-maven is discontinued and proposes an alternative method of accomplishing the same thing

I think what's appropriate is to merge as is and open a new ticket to refactor to remove this plugin altogether.

Reply:
opened STREAMS-525